recipes/raspa3/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "raspa3" %}
+{% set version = "v3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/iRASPA/RASPA3/archive/{{ version }}.tar.gz
+  sha256: 3a2d094bda0cbd48e8ca1760c73bfe70908ad3c154fc7b6f07003dbcad0dd8e4
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang-tools
+    - llvm-tools
+    - cmake
+    - ninja
+    - lld
+    - libcxx-devel  # [not win]
+  host:
+    - libcxx
+    - libcxxabi  # [linux]
+    - libgcc  # [linux]
+    - sysroot_linux-64  # [linux]
+    - hdf5
+    - openssl
+    - libcurl
+    - libaec
+    - lapack
+    - blas
+    - libopenblas
+    - zlib
+  run:
+    - sysroot_linux-64  # [linux]
+    - libcxxabi  # [linux]
+    - libgcc  # [linux]
+    - libcxx
+    - hdf5
+    - zlib
+    - openssl
+    - libcurl
+    - libaec
+    - lapack
+    - blas
+    - libopenblas
```

Comment:
```suggestion
```
All of these packages should automatically added by `run_exports` so long as they have been included in `host` requirements.